modules/backend/formwidgets/Repeater.php

```diff
@@ -202,16 +202,16 @@ protected function processSaveValue($value)
         /*
          * Give repeated form field widgets an opportunity to process the data.
          */
-        foreach ($this->formWidgets as $field => $form) {
-            foreach ($form->getFormWidgets() as $formField => $widget) {
-                $parts = HtmlHelper::nameToArray($field . '[' . $formField . ']');
-
-                $widgetValue = $widget->getSaveValue($this->dataArrayGet($value, $parts));
-                if (empty($widgetValue) || !count($widgetValue)) {
-                    continue;
-                }
-                $this->dataArraySet($value, $parts, $widgetValue);
```

Comment:
Should these helper functions be removed now too?

Reply:
Yep, removed :)